### PR TITLE
Recipes

### PR DESCRIPTION
**Pull request by sajibarafatsiddiqui**
group that has less than 3 members as I am solo now:
- Followed the layout of the wireframes provided. 
- Login page and registration page:
      built with Devise.
- Food list 
      display a list of food added by the logged-in user as in the wireframe (display also quantity of a given food).
      lead to a form that allows users to add new food.
- Recipes list:
     display a list of recipes created by the logged-in user as in the wireframe.
     lead to recipe details.
     if the user is the owner of the recipe, should allow the user to delete it.
- Public recipe list:
     display a list of all public recipes ordered by newest as in the wireframe.
     lead to recipe details.
- Recipe details:
     display a toggle button that allows for a recipe to be made public or private.
     recipe is public or the user is the owner of the recipe, should display the recipe details as in the wireframe.
     user is the owner of the recipe, should lead to the form that allows the user to add new food.
- General shopping list view
      show the list of food that is missing for all recipes of the logged-in user (compare the list of food for all recipes with the 
      general food list of that user).
- count the total food items and total price of the missing food.
- Made sure there are no N+1 queries happening.
-  navigation menu that allows users to open all of the pages you created.
- Write unit and integration tests
food.

### CODE REVIEW

**Comment by sajibarafatsiddiqui** [comment]
image

File: README.md [hunk 1]
```diff
@@ -1,24 +1,149 @@
...
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+# 📖 BlogMaster <a name="about-project"></a>
```
Comment (codepantha) [inline README.md]: - [ ] Please change the title of your project from **BlogMaster** to a more appropriate title.
- [ ] Kindly find and replace every occurence of **BlogMaster** in your README file.

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @sajibarafatsiddiqui 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [ ] The user registration feature is not working as a result of an `Unpermitted parameter` called `name`. In order to solve this error, you need to `configure_permitted_parameters`. Kindly use this [resource](https://www.appsloveworld.com/ruby/100/14/devise-unpermitted-parameters) to help you achieve it. 

This error also prevents me from checking the food list, recipes list, public recipes list, recipe details page, and general shopping list view.
Please note that the next reviewer will ensure that these project criteria are met 👍 
image
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sajibarafatsiddiqui) [reply README.md]: The issues have been resolved

**Comment by sajibarafatsiddiqui** [comment]
> image

please check the image as a proof of being solo.

**Review by andersonlebon — APPROVED** [review]
## Hi @sajibarafatsiddiqui 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**